ci/python_test.sh

```diff
@@ -12,10 +12,16 @@ fi
 
 BIN=$VENV/bin/
 
-$BIN/pip -v install --upgrade pip setuptools wheel
+source $BIN/activate
+pip -v install --upgrade pip setuptools wheel
 
 # install
-$BIN/pip -v install --force .
-$BIN/pip install nose
+pip -v install --force .
+pip install nose
 
-$BIN/nosetests -s -v python/tests
+nosetests -s -v -P python/tests
+
+PYTHON_MAJOR_VERSION=$(python -c 'import sys; print(sys.version_info[0])')
+if [[ $PYTHON_MAJOR_VERSION -ge 3 ]]; then
+    python setup.py test
```

Comment:
The python test would trigger both unittest and doc test. I don't know where in the package it sets sphinx > 2.0, the doc build always failed for python 2. So I skip the test for python 2.

Reply:
Seems reasonable to me. IMO, Python 2 support should be mostly best effort by now, anyways.